scripts/get_stress_test_summary.py

```diff
@@ -0,0 +1,62 @@
+import os
+import statistics
+from glob import glob
+from scripts.get_stats import get_stats
+
+
+def get_results(folders):
+
+    results = {
+        "get": [],
+        "post": [],
+        "average_get": [],
+        "average_post": [],
+        "average_total": [],
+        "error_percentage": 0,
+        "total_requests": 0,
+        "total_failures": 0
+    }
```

Comment:
This is repeated from the get_test module. Would be worth refactoring it out so that it's DRY. Also, I think a `typing.NamedTuple` or DataClass would work well here